packages/react-scripts/config/utils/ensureSlash.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function ensureSlash(path, needsSlash) {
```

Comment:
I would like to move this into `paths.js` and instead export `publicPath`, `publicUrl` and whatever else is needed from it. This would be more explicit IMO.